src/frontends/onnx/frontend/src/op/quantize_linear.cpp

```diff
@@ -231,19 +272,39 @@ ov::OutputVector quantize_linear(const ov::frontend::onnx::Node& node) {
     const auto zero_point = ai_onnx::detail::get_zero_point(inputs);
 
     // per-tensor quantization, axis attribute ignored
-    if (scale.get_partial_shape().rank().is_static() &&
-        (scale.get_partial_shape().rank().get_length() == 0 ||
-         (scale.get_partial_shape().rank().get_length() == 1 && scale.get_partial_shape()[0] == 1)) &&
-        zero_point.get_partial_shape().rank().is_static() &&
-        (zero_point.get_partial_shape().rank().get_length() == 0 ||
-         (zero_point.get_partial_shape().rank().get_length() == 1 && zero_point.get_partial_shape()[0] == 1))) {
+    if (ai_onnx::detail::is_per_tensor_quantization(scale, zero_point)) {
         return ai_onnx::opset_1::quantize_linear(node);
     }
-
-    return detail::quantize_linear(x, scale, zero_point, node.get_attribute_value<int64_t>("axis", 1), node);
+    return detail::quantize_linear(x, scale, zero_point, node.get_attribute_value<int64_t>("axis", 1), 0, node);
 }
-ONNX_OP("QuantizeLinear", OPSET_SINCE(13), ai_onnx::opset_13::quantize_linear);
+ONNX_OP("QuantizeLinear", OPSET_RANGE(13, 19), ai_onnx::opset_13::quantize_linear);
 }  // namespace opset_13
+
+namespace opset_21 {
+ov::OutputVector quantize_linear(const ov::frontend::onnx::Node& node) {
+    const ov::OutputVector inputs{node.get_ov_inputs()};
+
+    FRONT_END_GENERAL_CHECK(2 <= inputs.size() && inputs.size() <= 3,
```

Comment:
Please, use common::default_op_checks() like here: https://github.com/openvinotoolkit/openvino/blob/2f14b9ad14a241ab6908a0b11ba5c6d5d6ec3b48/src/frontends/onnx/frontend/src/op/com.microsoft/simplified_layer_normalization.cpp#L28

Reply:
This only checks minimum parameter counter. Here need to check the maximum parameter counter. So add new API to handle this.